command/commands/user.py

```diff
@@ -6,49 +6,58 @@
 class UserCommand:
     """Represent User Command Parser."""
 
-    __command_name = "user"
-    __help = "User Command Reference:\n\n @rocket user\n\n Options:\n\n" \
-             " edit \n --name NAME\n" \
-             " --email ADDRESS\n --pos YOURPOSITION\n" \
-             " --major YOURMAJOR\n --bio YOURBIO\n" \
-             " 'edit properties of your Launch Pad profile\n" \
-             " surround arguments with spaces with single quotes'" \
-             "\n ADMIN/TEAM LEAD ONLY option: --member MEMBER_ID\n" \
-             " 'edit properties of another user's Launch Pad profile'\n\n" \
-             " view MEMBER_ID\n 'view information about a user'\n\n " \
-             "help\n 'outputs options for user commands'\n\n " \
-             "ADMIN ONLY\n\n delete MEMBER_ID\n" \
-             " 'permanently delete member's Launch Pad profile'"
+    def __init__(self):
```

Comment:
In the future it would be better to put these changes in a different pull request, so they can be reviewed independently (though since it's already here, don't bother changing it for this case).

Reply:
Sounds good